### DISCUSSION TITLE
laser engraver stopt na een minuut

### DISCUSSION THREAD

**Discussion by Nico1958** [arkypita/LaserGRBL#871] [Q&A]
Ik heb een laser engraver gekocht bij Annoy tools en gebruik laser GRBL software v3.0.24
Als ik een project start dan stopt de laser al na een paar minuten met branden en moet ik hem handmatig weer starten. Kent iemand dit probleem? en wat is hier aan te doen?

**Comment by Schwarzi07** [comment]
Have you enabled automatic cooling in the LaserGRBL settings?
